### DISCUSSION TITLE
[Question][Hw1]

### DISCUSSION THREAD

**Discussion by rzhsynv** [prosyslab-classroom/cs348-information-security#674] [Q&A]
Name: Rza Huseynov

Hi. I have a question about solving Vigenere's crack algorithm. Is it okay if I share it from now or should I wait until Friday (late deadline) to ask a specific solution about solutions? Also will you share the solutions for the HW?

**Comment by KAIST-JongchanPark** [comment]
1. Please wait until late deadline. However, please just share the idea for crack, not the code. (someone can retake this course, ...)
2. We will not share the solution.

**Comment by rzhsynv** [comment]
Alright, I will share my idea only after the late deadline.
